pkg/apis/v1/taints.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import v1 "k8s.io/api/core/v1"
+
+// Karpenter specific taints
+const (
+	DisruptionTaintKey             = Group + "/disruption"
```

Comment:
Are these taints changing? If so, should we match them to what we want them to be at v1?

Reply:
Consider syncing and closing on this with @jmdeal

Reply:
Yeah, I can sync up with him. However, I'm planning to add in to change in separate PRs